app/assets/javascripts/galleries/add_new.js

```diff
@@ -117,6 +123,7 @@ function cleanUpRows() {
         anySet = true;
         return false;
       }
+      return false;
```

Comment:
Changes the behavior – $.each stops when you return false, and we want to intentionally return undefined (or true if we really need to return something) to make sure it continues.

Reply:
*huh*. Okay.